docs/products/clickhouse/howto/connect-with-php.rst

```diff
@@ -0,0 +1,71 @@
+Connect to the Aiven for ClickHouse® service with PHP
+=====================================================
+
+Learn how to connect to your Aiven for ClickHouse® service with PHP using the PHP ClickHouse client and the HTTPS port.
+
+Pre-requisites
+--------------
+
+* `PHP 7.4 or later <https://www.php.net/downloads>`_
+* ``smi2/phpclickhouse`` library
+* `Composer <https://getcomposer.org/>`_
+
+.. tip::
+
+    You can install ``smi2/phpclickhouse`` with the following command:
+
+    .. code-block:: php
+
+        composer require smi2/phpclickhouse
+
+    or
+
+    .. code-block:: php
+
+        php composer.phar require smi2/phpclickhouse
+
+Identify connection information
+-------------------------------
+
+To run the code for connecting to your service, first identify values of the following variables:
+
+===========================     =======================================================================================
+Variable                        Description
+===========================     =======================================================================================
+``https``                       Required to be set to ``true``
+``host``                        ``Host`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``port``                        ``Port`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``username``                    ``User`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``password``                    ``Password`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``database``                    ``Database Name`` in the ClickHouse service available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``setTimeout``                  Supports only integer values, measured in seconds
+``setConnectTimeout``           Supports only integer values, measured in seconds
+``ping``                        Set to ``true`` to enable throwing an exception if a connection cannot be established
```

Comment:
Sorry I missed these, they should be removed since they are not used in the example.